### PR TITLE
Use react-icons component for rendering individual icons

### PR DESCRIPTION
**Pull request by anshumanv**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -246,26 +249,35 @@ class App extends Component {
 
         <div className="bottom-bar">
           <div className="bottom-left">
-            <div id="previous" onClick={this.previousTrack} title="Play Last">
-              <i className="fa fa-backward fa-2x" />
+            <div title="Play Previous">
+              <TiArrowLeftThick
+                onClick={this.previousTrack}
```
Comment (filoxo) [inline src/components/App.js]: Don't add the click handler on the actual icon. This has the same implications of adding it to a div.

File: src/components/App.js [hunk 2]
```diff
@@ -246,26 +249,35 @@ class App extends Component {
...
+                size={35}
```
Comment (filoxo) [inline src/components/App.js]: Since this value is reused for all the icons, would it make sense to move the value into `constants.js` instead?

**Review by filoxo — CHANGES_REQUESTED** [review]

File: src/components/App.js (hunk 2 above)
Comment (anshumanv) [reply src/components/App.js]: Yes, on it.

File: src/components/App.js (hunk 1 above)
Comment (anshumanv) [reply src/components/App.js]: Okay, changing.

**Comment by anshumanv** [comment]
@filoxo please review again.

**Comment by anshumanv** [comment]
@filoxo When I'm trying to add a border to all the icons, getting something like this - 
image
Can you help?

**Comment by filoxo** [comment]
I'll have a look at this later today.

**Comment by anshumanv** [comment]
Sure thanks @filoxo :smile:

**Pull request merged**